Cargo.toml

```diff
@@ -34,7 +34,7 @@ async-stream = "0.3"
 async-trait = "0.1"
 bstr = "1.8"
 dns-parser = "0.8.0"
-err-derive = "0.2"
+thiserror = "1"
```

Comment:
Why not bump to 2.0?

Reply:
because hub-core is still using 1.0 not 2.0 and I didn't want _another_ duplicate dependency in the allow as part of the patch.